ibis/backends/base/sql/alchemy/translator.py

```diff
@@ -69,17 +69,24 @@ def _reduction(self, sa_func, expr):
         ):
             arg = arg.cast(dt.Int32(nullable=type.nullable))
 
+        argtuple = tuple(
+            arg
+            for argname in op.argnames
+            if argname != "where"
+            if isinstance(arg := getattr(op, argname), ir.Expr)
```

Comment:
`arg for name, arg in zip(op.argnames, op.args) if isinstance(arg, ir.Expr) and name != "where"`